### PR TITLE
[MIG] academic/: Migration to 13.0

### PR DESCRIPTION
**Pull request by nmadhoc**

### CODE REVIEW

File: academic/security/academic_security.xml [hunk 1]
```diff
@@ -110,17 +64,16 @@
...
         <field name="name">Teacher's Portal</field>
-        <field name="is_portal" eval="True"/>
-        <field name="implied_ids" eval="[(4, ref('group_portal_student')),(4, ref('evaluation.group_survey_see_scores'))]"/>
+        <field name="implied_ids" eval="[(4, ref('group_portal_student'))]"/>
         <field name="category_id" ref="base.module_category_user_type"/>
         <field name="comment">Teacher's Portal members have specific access rights (such as record rules and restricted menus).
                 They usually do not belong to the usual odoo groups.</field>
     </record>
 
+
```
Comment (nicomacr) [inline academic/security/academic_security.xml]: Remove this blank space

File: academic/security/ir.model.access.csv [hunk 2]
```diff
@@ -9,46 +9,21 @@ access_academic_subject_manager,academic.subject.manager,model_academic_subject,
 access_academic_subject_user,academic.subject.user,model_academic_subject,group_user,1,1,1,1
 access_academic_group_evaluation_manager,academic.group_evaluation.manager,model_academic_group_evaluation,group_manager,1,1,1,1
 access_academic_group_evaluation_user,academic.group_evaluation.user,model_academic_group_evaluation,group_user,1,1,1,1
-access_survey_survey_manager,survey.survey.manager,model_survey_survey,group_manager,1,1,1,1
-access_survey_survey_user,survey.survey.user,model_survey_survey,group_user,1,1,1,1
-access_survey_user_input_manager,survey.user_input.manager,model_survey_user_input,group_manager,1,1,1,1
-access_survey_user_input_user,survey.user_input.user,model_survey_user_input,group_user,1,1,1,1
-access_academic_observation_category_manager,academic.observation_category.manager,model_academic_observation_category,group_manager,1,1,1,1
-access_academic_observation_category_user,academic.observation_category.user,model_academic_observation_category,group_user,1,1,1,1
+
```
Comment (nicomacr) [inline academic/security/ir.model.access.csv]: same as the previous comment

File: academic/views/academic_menuitem.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@
     <menuitem name="Academic" id="menu_academic" sequence="20" groups="group_user" web_icon="academic,static/description/icon.png"/>
     <menuitem name="Configuration" id="menu_configuration" sequence="500" parent="menu_academic" groups="group_manager" />
     <menuitem name="Groups" id="menu_data" sequence="20" parent="menu_academic" groups="group_user" />
-    <menuitem name="Evaluations" id="menu_evaluations_main" sequence="30" parent="menu_academic" groups="survey.group_survey_manager" />
+
```
Comment (nicomacr) [inline academic/views/academic_menuitem.xml]: remove this line

File: academic/views/res_partner_views.xml [hunk 4]
```diff
@@ -17,12 +17,7 @@
             <field name="user_id" position="attributes">
                 <attribute name="groups">base.group_no_one</attribute>
             </field>
-            <filter name="customer" position="attributes">
-                <attribute name="groups">base.group_no_one</attribute>
-            </filter>
-            <filter name="supplier" position="attributes">
-                <attribute name="groups">base.group_no_one</attribute>
-            </filter>
+
```
Comment (nicomacr) [inline academic/views/res_partner_views.xml]: remove line

File: oca_dependencies.txt [hunk 5]
```diff
@@ -1,3 +1,2 @@
-ingadhoc-survey https://github.com/ingadhoc/survey.git
-ingadhoc-website https://github.com/ingadhoc/website.git
 ingadhoc-miscellaneous https://github.com/ingadhoc/miscellaneous.git
+
```
Comment (nicomacr) [inline oca_dependencies.txt]: remove line

File: academic/demo/res_users_demo.xml [hunk 6]
```diff
@@ -29,7 +29,7 @@
         <field name="login">lsali</field>
         <field name="email">lsali@academic.com</field>
         <field name="password">lsali</field>
-        <field name="company_id" ref="res_company_los_molinos"/>
+        <field name="company_id" ref="base.main_company"/>
```
Comment (nicomacr) [inline academic/demo/res_users_demo.xml]: why change the company, the old one dosen't exist any more?

File: academic/demo/res_company_demo.xml [hunk 7]
```diff
@@ -18,6 +18,7 @@
         <field name="report_header">ASOCIACIÓN PARA LA PROMOCIÓN DEPORTIVA, EDUCATIVA Y SOCIAL</field>
         <!-- <field name="partner_id" ref="main_partner"/> -->
         <field name="currency_id" ref="base.ARS"/>
+        <!-- <field name="fiscalyear_last_day">31</field> -->
```
Comment (nicomacr) [inline academic/demo/res_company_demo.xml]: Apply this line and the lines below:

Why the comment lines?

It's not use, better delete them

File: academic/controllers/website.py [hunk 8]
```diff
@@ -1,33 +0,0 @@
-from odoo import http
-from odoo.addons.website.controllers.main import Website
```
Comment (nicomacr) [inline academic/controllers/website.py:2]: Why delete this file?

It's necessary to redirect portal user to backend

**Review by nicomacr — CHANGES_REQUESTED** [review]
Hello @nmadhoc 

I left some comments in the code for you to check out.

Bes,

**Comment by nmadhoc** [comment]
> Hello @nmadhoc
> 
> I left some comments in the code for you to check out.
> 
> Bes,

Thanks so much for the feedback @nicomacr!  
I recently uploaded the changes you requested and the tests are successful.
Take care!

**Review by nicomacr — DISMISSED** [review]
Technical Review +1

**Pull request merged**